src/applications/claims-status/containers/StemClaimStatusPage.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { setUpPage } from '../utils/page';
+
+import { Link } from 'react-router';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
+import StemAskVAQuestions from '../components/StemAskVAQuestions';
+import ClaimsBreadcrumbs from '../components/ClaimsBreadcrumbs';
+import ClaimsUnavailable from '../components/ClaimsUnavailable';
+import StemDeniedDetails from '../components/StemDeniedDetails';
+
+class StemClaimStatusPage extends React.Component {
+  componentDidMount() {
+    this.setTitle();
+    setUpPage();
+  }
+
+  setTitle() {
+    document.title =
+      'Status - Your Edith Nourse Rogers STEM Scholarship Application Claim';
+  }
+
+  render() {
+    const { claim, loading } = this.props;
+    const claimsPath = `your-stem-claims/${this.props.params.id}`;
+    let content;
+    if (loading) {
+      content = (
+        <LoadingIndicator
+          setFocus
+          message="Loading your claim information..."
+        />
+      );
+    } else if (claim) {
+      content = (
+        <StemDeniedDetails
+          deniedAt={claim.attributes.deniedAt}
+          isEnrolledStem={claim.attributes.isEnrolledStem}
+          isPursuingTeachingCert={claim.attributes.isPursuingTeachingCert}
+        />
+      );
+    } else {
+      content = (
+        <>
+          <h1>We encountered a problem</h1>
```

Comment:
I'm not sure if this shouldn't be an `h2`, while keeping your previous `h1` content... Please refer to your accessibility specialist.